src/components/progress-circle/progress-circle.scss

```diff
@@ -0,0 +1,110 @@
+@import "variables";
+
+@import "default-theme";
+
+/* Animation Durations */
+$md-progress-circle-duration : 5.25s !default;
+$md-progress-circle-value-change-duration : $md-progress-circle-duration * (1 / 4) !default;
+$md-progress-circle-constant-rotate-duration : $md-progress-circle-duration * (5 / 9) !default;
+$md-progress-circle-sporadic-rotate-duration : $md-progress-circle-duration !default;
+
+/** Component sizing */
+$md-progress-circle-stroke-width: 10px !default;
+$md-progress-circle-radius: 40px !default;
+$md-progress-circle-circumference: $pi * $md-progress-circle-radius * 2 !default;
+$md-progress-circle-center-point: 50px !default;
+// Height and weight of the viewport for md-progress-circle.
+$md-progress-circle-viewport-size : 100px !default;
+
+
+:host {
+  display: block;
+  /** Height and width are provided for md-progress-circle to act as a default.
+      The height and width are expected to be overwritten by application css. */
+  height: $md-progress-circle-viewport-size;
+  width: $md-progress-circle-viewport-size;
+
+  /** SVG's viewBox is defined as 0 0 100 100, this means that all SVG children will placed
+      based on a 100px by 100px box.
+
+      The SVG and Circle dimensions/location:
+        SVG
+          Height: 100px
+          Width: 100px
+        Circle
+          Radius: 40px
+          Circumference: 251.3274px
+          Center x: 50px
+          Center y: 50px
+  */
+  svg {
+    height: 100%;
+    width: 100%;
+  }
+
+
+  circle {
+    cx: $md-progress-circle-center-point;
+    cy: $md-progress-circle-center-point;
+    fill: transparent;
+    r: $md-progress-circle-radius;
+    stroke: md-color($md-primary, 100);
```

Comment:
I was not sure what hue that we wanted, and then forgot to find out.

What hue are you expecting here?